readthedocs/config/config.py

```diff
@@ -670,6 +670,13 @@ class BuildConfigV2(BuildConfigBase):
         'dirhtml': 'sphinx_htmldir',
         'singlehtml': 'sphinx_singlehtml',
     }
+    valid_os = ['ubuntu-20']
+    valid_languages = {
+        'python': ['3.9', '3.8'],
+        'nodejs': ['13', '14'],
+        'rust': ['1.54'],
+        'golang': ['1.17'],
+    }
```

Comment:
@humitos are these static values? I guess the list of available languages at least is.

Reply:
This list is static, yes.

There are some values missing here for the Python key, like:

* `2.7`
* `pypy3.7`
* `miniconda3-4.7`
* `mambaforge-4.10`

For node, we should allow `14` (LTS) and `16` (latest)